### PR TITLE
Fixed responsiveness of inputs and modal

### PR DESCRIPTION
**Pull request by kajsaeggum**
# Describe your changes
Some small adjustments to components to make them more responsive when screen size changes:
1. Changed width settings of select, autocomplete and multiselect elements to max-width instead of width, so they can go smaller than that if there's less space. To fix situations like these (MdSelect-component seen here on mobile screen):
image

2. Added "small"-width to MdInput outer wrapper. Even though I use MdInput with size="small" here, it still takes up all that extra width because the outer wrapper didn't have a width restriction like the inner wrapper does when size is set to small:
image

3. We needed a modal in our form, that doesn't change size according to it's content because the content changes a lot when used so the modal would "jump around" both horizontally and vertically as the user was typing in it. I added a "small"-setting to MdModal, which have a fixed vh/vw size regardless of its content.

Otherwise our modal would start out small like this, before the user searched for anything:
image

And then when the search result was present it would expand to this:
image

And if the user kept writing in the search field to the point where the search yielded no results it would shrink again to:
image

## Checklist before requesting a review

- [x] I have performed a self-review and test of my code
- [x] I have added label to the PR (`major`, `minor` or `patch`)
- [ ] If new component: Is story for component created in `stories`-folder?
- [ ] If new component: Is tsx-file import added to `packages/react/index.tsx`?
- [ ] If new component: Is css-file added to `packages/css/index.css`?

### CODE REVIEW

**Review by aurorascharff — APPROVED** [review]
Ser bra ut :)

File: packages/css/src/modal/modal.css [hunk 1]
```diff
@@ -40,6 +40,14 @@
   box-shadow: 8px 8px 16px var(--mdGreyColor60);
 }
 
+.md-modal__inner-wrapper--small {
+  width: 40vw;
```
Comment (thomaslarsson) [inline packages/css/src/modal/modal.css:44]: Gir det mening å innføre en min-size på modaler generelt for å forbedre dette problemet generelt, eller er problemstillingen forbeholdt small size?

Comment (kajsaeggum) [reply packages/css/src/modal/modal.css:44]: Jeg tenkte at det godt kan finnes caser hvor det er riktig at modalen tilpasser størrelse til innehold, slik det er i dag. Så jeg ville ikke ødelegge det noe, men heller utvide med mulighet for en fast størrelse med opt-in. 
Og hvis det viser seg at det er flere caser med slikt behov så går det jo ann å gjøre den funksjonaliteten enda mer sofistikert med flere størrelser, f.eks. small, medium, large. Eller rett og slett custom ved å spesifisere ønsket bredde/høyde som input eller noe slikt

Comment (thomaslarsson) [reply packages/css/src/modal/modal.css:44]: Nice. Da har du tenkt tanken i hvert fall. Jeg var btw på vei til å godkjenne i det du merget, men tenkte jeg bare skulle lufte tanken. Bra jobba. :)

Comment (kajsaeggum) [reply packages/css/src/modal/modal.css:44]: Snakket litt mer med Aurora om det nå, kommer en ny pr asap med forslag som gjør det mer customizable :)

**Pull request merged**